rs/bitcoin/ckbtc/minter/ckbtc_minter.did

```diff
@@ -29,6 +29,7 @@ type DefiniteCanisterSettings = record {
   reserved_cycles_limit : nat;
   log_visibility: LogVisibility;
   wasm_memory_limit : nat;
+  wasm_memory_threshold : nat;
```

Comment:
Here for a lack of a better place: could you add a PR description to motivate this change?

Reply:
@gregorydemay It seems Bitcoin and Ethereum tests are failing because of the incompatible candid interface, but I do not know what should be changed. Do you have any idea how to fix it?

Reply:
> I just reverted changes on your side, in the case you want to use the feature, you can add the new field, for now, I believe it is unnecessary.

It would be nice to have it is this is exposed. Do you have the exact error that occurred? We are working on that area, so it could be that the error is due to an outdated master (merging master should fix it)